processing/src/test/java/io/druid/query/metadata/SegmentMetadataQueryTest.java

```diff
@@ -1077,6 +1081,33 @@ public Interval getInterval()
     for (int i = 0; i < filteredSegments2.size(); i++) {
       Assert.assertEquals(expectedSegments2.get(i).getInterval(), filteredSegments2.get(i).getInterval());
     }
+
```

Comment:
@leventov added the `isUsingDefaultInterval` in here along with a failing test (because of filterSegments implementation). What should I do about filterSegments though ?